OpenKeychain/src/main/java/org/sufficientlysecure/keychain/securitytoken/usb/UsbTransport.java

```diff
@@ -211,36 +230,36 @@ public TransportType getTransportType() {
     @Override
     public TokenType getTokenType() {
         switch (usbDevice.getVendorId()) {
-            case 4176: {
+            case VID_YUBIKEY: {
```

Comment:
How about `VENDOR_` and `DEVICE_`? I find particularly `PID` misleading

Reply:
👍 I changed to ``VENDOR`` and ``PRODUCT`` as in the terminology of USB.